toothpick-runtime/src/main/java/toothpick/ScopeNode.java

```diff
@@ -200,6 +197,14 @@ public boolean isBoundToScopeAnnotation(Class<? extends Annotation> scopeAnnotat
     return scopeAnnotationClasses.contains(scopeAnnotationClass);
   }
 
+  /**
+   * Resets the state of the scope
```

Comment:
missing period at the end of javadoc. You will burn in hell. 
Add this is for testing only.